app/src/main/java/com/infomaniak/swisstransfer/ui/navigation/NavigationDestination.kt

```diff
@@ -110,48 +111,96 @@ sealed class MainNavigation : NavigationDestination() {
  * Sealed class representing the navigation arguments for the new transfer flow.
  */
 @Serializable
-sealed class NewTransferNavigation : NavigationDestination() {
+sealed class NewTransferNavigation(override val matomoValue: String) : NavigationDestination() {
 
     @Serializable
-    data object ImportFilesDestination : NewTransferNavigation()
+    data object ImportFilesDestination : NewTransferNavigation("ImportFileView")
```

Comment:
This destination should soon change after @LouisCAD 's PR. Would we want to already use a name that matches the new screen name so we don't send incoherent values to matomo?

Reply:
Now that we have the need to name each destination, we should align with iOS, that's why we should use this name, and not the current nor future class's name

Reply:
After talking about it in real, the problem is not the fact that he destination will change, it's that some destination will totally be erased